lib/Jit/LLILCJit.cpp

```diff
@@ -303,6 +283,16 @@ CorJitResult LLILCJit::compileMethod(ICorJitInfo *JitInfo,
 
       // Tell the CLR that we've successfully generated code for this method.
       Result = CORJIT_OK;
+
+      if (Context.Options->DumpLevel >= ::DumpLevel::SUMMARY) {
+         errs() << "LLILC compiled: " 
+                << ", Entry = " << *NativeEntry 
+                << ", End = " << (*NativeEntry + *NativeSizeOfCode)
+                << ", size = " << *NativeSizeOfCode 
+                << " method = " << Context.MethodName 
+                << '\n';
+      }
+
```

Comment:
> When debugging I often want to know if a location is in code we jitted and this
give that capability.

OK. FWIW, the sos extension for WinDBG already gives you this capability via the `!ip2md` command.

I wonder if a new dump level--say DEBUG--would be useful here to avoid cluttering the logs in the typical case. No strong feelings, however.